### DISCUSSION TITLE
Error Messages - Column V

### DISCUSSION THREAD

**Discussion by pmj-mm** [CMSgov/hospital-price-transparency#120] [Q&A] [answered]
I am receiving error messages for column V on all lines with blank code types. 

Based on CMS instructions: If you would like to include an explanation for the blanks, you may do so using Additional Generic Notes or Additional Payer-Specific Notes.”   I have a note in column U explaining the blank codes but no data in column V. 
What would be causing this?

Thank You

**Comment by RidenourK** [comment, marked as answer]
@pmj-mm. Thank you for your question. 

Please refer to the conditional requirements in the [CSV documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV) on this GitHub repository, which indicate:

The following conditional requirements must be met for an MRF to be considered valid. These conditional requirements enforce regulatory rules for required data elements, provide flexibility in the development of MRFs, and ensure corresponding information is encoded for items and services to be understandable by the end user.

If a standard charge is encoded, there must be a corresponding code and code type pairing. The code and code type pairing do not need to be in the first code and code type columns (i.e., code|1 and code|1|type).

If a value is encoded in the "code", a value must also be encoded in the corresponding "code type". Conversely, if a value is encoded in the "code type", a value must be encoded in the corresponding "code" (e.g., if a value is encoded in code|2|type, then a value must be encoded in code|2).
